### DISCUSSION TITLE
GeoTiff on mobile platform

### DISCUSSION THREAD

**Discussion by GreenleafDk** [Mapsui/Mapsui#2110] [Q&A]
It seems that reading a geoTiff layer on Android using Xamarin is not supported.
Are there any special reasons for this, or would it be something I could work around somewhat easily?

**Comment by inforithmics** [comment]
I tried it out in Xamarin.Forms Android and it worked. you have to add Mapsui.Extensions for geotiff layer support.
https://www.nuget.org/packages/Mapsui.Extensions

**Reply by GreenleafDk** [reply]
Thanks for the hint. I am trying it out but it forces an update to mapsui 4.0 which apparently changed a bunch of namespaces, leading to unresolved references, that I need to figure out first. That you made geotiff work on android is very promissing, though. Hope to also get it moving soon now.
